src/lib/Server/Output/ValueObjectVisitor/RestContent.php

```diff
@@ -50,53 +50,45 @@ public function visit(Visitor $visitor, Generator $generator, $data)
         $visitor->setHeader('Content-Type', $generator->getMediaType($mediaType));
         $visitor->setHeader('Accept-Patch', $generator->getMediaType('ContentUpdate'));
 
-        $generator->startAttribute(
+        $generator->attribute(
             'href',
             $data->path === null ?
                 $this->router->generate('ibexa.rest.load_content', ['contentId' => $contentInfo->id]) :
                 $data->path
```

Comment:
While at it let's improve readability:
```suggestion
            $data->path ?? $this->router->generate(
                'ibexa.rest.load_content',
                ['contentId' => $contentInfo->id]
            )
```